### ISSUE TITLE
fashionMNIST results

### ISSUE THREAD

**Issue by bkj** [ajbrock/FreezeOut#3]
This code is linked from the [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) repo, w/ very good results.  Do you have a script somewhere I might be able to use to reproduce those numbers?  

Thanks
Ben

**Comment by ajbrock** [comment]
Just drop a FashionMNIST data-loader in [utils.py](https://github.com/ajbrock/FreezeOut/blob/master/utils.py#L71) and replace the mean / average statistics. Run it with the options that turn FreezeOut off, something like:
`python train.py --t_0 1.0 --epochs 300 --which_dataset 10`

and then whatever args you want for your model of choice. The scripts I used are on another machine I won't have access to for a few months, sorry.

Also, my results are not very good, it's just that no one has ever bothered to properly benchmark that dataset. With a tiny bit of hyperparameter tuning I wouldn't be surprised if you could exceed 98% accuracy, modulo how many mislabeled samples there are in the test set.

**Comment by bkj** [comment]
OK thanks.  Do you have any pointers to a well-tuned model? 

~ Ben

**Comment by ajbrock** [comment]
Probably try one of those shake-shake or shake-drop variants, with a well-tuned SGDR cycle and distillation a-la born again neural nets or stochastic weighted averaging, and figure out the right width/depth. Googling any of those terms should point you in the right direction if they're not familiar =)

**Comment by bkj** [comment]
Yeah I bet that would do it... but have you ever seen a model that uses all of those techniques? I'm not sure _any_ datasets are properly benchmarked by your definition! :)

Related -- have you ever seen any code implementing born again networks?  It's fairly simple, but details can be finnicky.

~ Ben

**Comment by ajbrock** [comment]
Hah, I'm not saying that datasets that haven't been hit with the latest-and-greatest aren't properly benchmarked, but no one's beaten a vanilla WRN40-4 with standard data aug. Kagglers would have a field day =p

As to BANN you can try asking Tommaso but i'm not familiar with any implementations.
